salt/modules/file.py

```diff
@@ -4459,7 +4459,10 @@ def check_perms(name, ret, user, group, mode, attrs=None, follow_symlinks=False)
     is_dir = os.path.isdir(name)
     is_link = os.path.islink(name)
     if not salt.utils.platform.is_windows() and not is_dir and not is_link:
-        lattrs = lsattr(name)
+        try:
+            lattrs = lsattr(name)
+        except SaltInvocationError:
+            lsattrs = None
```

Comment:
lattrs is mis-spelled in this line.   if the except condition fires, then following `if lattrs is not None:` line will throw an `local variable 'lattrs' referenced before assignment` error.

Reply:
can you open a pr for this?